src/files.rs

```diff
@@ -136,6 +137,21 @@ impl Files {
     Ok(Response::new(Body::from(body.into_string())))
   }
 
+  fn download_icon(size: u8) -> PreEscaped<String> {
```

Comment:
Does Maud have an HTML type, that's more specific than `PreEscaped<String>`?

Reply:
No, not really. This is just what the `html!` macro returns. It has this type alias, though: https://docs.rs/maud/0.22.2/maud/type.Markup.html. I'll switch to that.